extracters/aata_data.py

```diff
@@ -82,27 +86,104 @@ def make_aata_article_dict(e):
 
 	This information is returned in a single `dict`.
 	'''
+
+	data = _xml_extract_article(e)
+	aata_id = data['_aata_record_id']
+	organizations = list(_xml_extract_organizations(e, aata_id))
+	authors = list(_xml_extract_authors(e, aata_id))
+	abstracts = list(_xml_extract_abstracts(e, aata_id))
+
+	data.update({
+		'_organizations': list(organizations),
+		'_authors': list(authors),
+		'_abstracts': list(abstracts),
+	})
+	return data
+
+def _xml_extract_article(e):
+	'''Extract information about an "article" record XML element'''
 	doc_type = e.findtext('./record_desc_group/doc_type')
 	title = e.findtext('./title_group[title_type = "Analytic"]/title')
+	var_title = e.findtext('./title_group[title_type = "Analytic"]/title_variant')
 	translations = list([t.text for t in
 		e.xpath('./title_group[title_type = "Analytic"]/title_translated')])
-	languages = set([t.text for t in e.xpath('./notes_group/lang_doc|./notes_group/lang_summary')])
+
+	doc_langs = {t.text for t in e.xpath('./notes_group/lang_doc')}
+	sum_langs = {t.text for t in e.xpath('./notes_group/lang_summary')}
+
+	isbn10 = [(t.text, isbn10Identifier) for t in e.xpath('./notes_group/isbn_10')]
+	isbn13 = [(t.text, isbn13Identifier) for t in e.xpath('./notes_group/isbn_13')]
+	issn = [(t.text, issnIdentifier) for t in e.xpath('./notes_group/issn')]
+
 	aata_id = e.findtext('./record_id_group/record_id')
-	organizations = list(_xml_extract_organizations(e, aata_id))
-	authors = list(_xml_extract_authors(e, aata_id))
-	abstracts = list(_xml_extract_abstracts(e, aata_id))
 	uid = 'AATA-%s-%s-%s' % (doc_type, aata_id, title)
 
+	classifications = []
+	title_type = model.Type(ident='http://vocab.getty.edu/aat/300055726', label='Title') # TODO: is this the right aat URI?
+	code_type = None # TODO: is there a model.Type value for this sort of code?
+	for cg in e.xpath('./classification_group'):
+		# TODO: there are only 61 unique classifications in AATA data; map these to UIDs
+		cid = cg.findtext('./class_code')
+		label = cg.findtext('./class_name')
+
+		name = model.Name()
+		name.classified_as = title_type
```

Comment:
This can just be a vanilla Name without a classification. It's not a title of a work.